### PR TITLE
V3

### PR DESCRIPTION
**Pull request by yashh-d**
Updated documentation to include sector, docket file number, and sec release number

### CODE REVIEW

File: RegalyticsRegulatoryArticle.cs [hunk 1]
```diff
@@ -84,6 +90,9 @@ public class RegalyticsRegulatoryArticle : BaseData
         [JsonProperty(PropertyName = "agencies")]
         public List<string> Agencies { get; set; }
 
+        [JsonProperty(PropertyName = "sector_type")]
+        public List<string> Sector { get; set; }
```
Comment (AlexCatarino) [inline RegalyticsRegulatoryArticle.cs]: The sector_type is a list of key-value pair (dictionary):
```
"sector_type": [{"name": "Financial"}, {"name": "Real Estate and Construction"}],
```

```csharp
public List<Dictionary<string,string>> Sector { get; set; }
```

**Review by AlexCatarino — CHANGES_REQUESTED** [review]
Thank you for your contribution, @yprime7!

The tests are not compiling. Please [compile and run the unit tests](https://www.quantconnect.com/docs/v2/lean-engine/contributions/datasets/testing-data-models#03-Run-Unit-Tests) before submitting the pull request. 

The sector type is not a list of string. When the reader parse the data, it throws an exception. Please [run the demonstration algorithm](https://www.quantconnect.com/docs/v2/lean-engine/contributions/datasets/testing-data-models#02-Run-Demonstration-Algorithms).

File: RegalyticsRegulatoryArticle.cs (hunk 1 above)
Comment (yashh-d) [reply RegalyticsRegulatoryArticle.cs]: Sounds good, just fixed the issues. Thanks for the links they were very helpful

**Review by AlexCatarino — APPROVED** [review]
Thank you, @yprime7 !

**Pull request merged**